### PR TITLE
added as_ptr to Runtime

### PR DESCRIPTION
**Pull request by Leechael**

### CODE REVIEW

File: qjsbind/src/engine.rs [hunk 1]
```diff
@@ -186,7 +186,7 @@ impl Drop for Context {
 }
 
 pub struct Runtime {
-    ptr: NonNull<c::JSRuntime>,
+    pub(crate) ptr: NonNull<c::JSRuntime>,
```
Comment (kvinwang) [inline qjsbind/src/engine.rs]: Since `fn as_ptr` resides in the same module, this modification is not neccessary.

File: qjsbind/src/engine.rs [hunk 2]
```diff
@@ -309,6 +309,12 @@ impl Runtime {
         data.abort_tx = Some(tx);
         rx
     }
+
+    pub fn as_ptr(&self) -> *mut c::JSRuntime {
+        self.ptr.as_ptr()
+    }
+
+
```
Comment (kvinwang) [inline qjsbind/src/engine.rs]: **By the way, consider running "Format Document" in VSCode when made changes to a file.**

```suggestion

    pub fn as_ptr(&self) -> *mut c::JSRuntime {
        self.ptr.as_ptr()
    }
```

Comment (Leechael) [reply qjsbind/src/engine.rs]: I'm using neovim. Which formatter should I use?

Comment (Leechael) [reply qjsbind/src/engine.rs]: I did format with `rustfmt` and force push, pls check.

**Comment by kvinwang** [comment]
LGTM

**Pull request merged**